prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -405,7 +405,7 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
             if (restUrl.contains("dataingestion.datateam-cdc-nbs")) {
                 val idTokenInfoString: String = client.post(restUrl) {
                     val credentialString = credential.user + ":" + credential.pass
-                    val basicAuth = "Basic " + Base64.encodeBytes(credentialString.encodeToByteArray())
+                    val basicAuth = "Basic " + Base64.getEncoder().encodeToString(credentialString.encodeToByteArray())
```

Comment:
Depending on Java standard library is always a good idea simple operations like this 👍

Reply:
Thank you @jalbinson